### PR TITLE
{bio}[intel/2019b] Pychopper v2.3.1 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/37392166b56f491a1ac1714d34c2be11 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3127.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/b2796880b37e02f45c396d8bfe8109f3 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 1 out of 6 (1 easyconfigs in this PR)
node150.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/f0dc452c23b94251b07dad695a9bd9eb for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in this PR)
node150.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/4b2b90374d91db55a4862abfb34c56ca for a full test report.

File: easybuild/easyconfigs/p/Pychopper/Pychopper-2.3.1-intel-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        'source_urls': ['https://github.com/nanoporetech/pychopper/archive'],
+        'checksums': ['ef92553c27a068baf18794a802c79fcc85554df4c9eec9da5e2e017dd6827e0a'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
```
Comment (lexming) [inline easybuild/easyconfigs/p/Pychopper/Pychopper-2.3.1-intel-2019b-Python-3.7.4.eb:49]: These `sanity_check_paths` can be removed as they are default in a PythonBundle
```suggestion
```

**Review by lexming — DISMISSED** [review]
Works well, I would just make the following minor change

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/branfosj/cc0202ba471e3ad94b3a1e146db44175 for a full test report.

File: easybuild/easyconfigs/p/Pychopper/Pychopper-2.3.1-intel-2019b-Python-3.7.4.eb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+toolchain = {'name': 'intel', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),  # for pandas
+    ('matplotlib', '3.1.1', versionsuffix),
+    ('Biopython', '1.75', versionsuffix),
+    ('parasail', '2.4.1'),
+    ('Seaborn', '0.10.0', versionsuffix),
```
Comment (branfosj) [inline easybuild/easyconfigs/p/Pychopper/Pychopper-2.3.1-intel-2019b-Python-3.7.4.eb:18]:
```suggestion
    ('python-parasail', '1.2', versionsuffix),
    ('Seaborn', '0.10.0', versionsuffix),
    ('tqdm', '4.41.1'),
```

**Review by branfosj — CHANGES_REQUESTED** [review]
Use existing `python-parasail` and `tqdm` deps.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/branfosj/d5f4a1d5b595647071ba33434691563f for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/branfosj/ccc2311b1021f4caca27245146bb9483 for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**